nfs-ganesha/build/build_rpm

```diff
@@ -117,12 +118,12 @@ if [ "$THROWAWAY" = false ] ; then
 }
 EOF
     # post the json to repo-extra json to chacra
-    curl -X POST -H "Content-Type:application/json" --data "@$WORKSPACE/repo-extra.json" -u $CHACRACTL_USER:$CHACRACTL_KEY ${chacra_url}/repos/${chacra_endpoint}/extra/
+    curl -X POST -H "Content-Type:application/json" --data "@$WORKSPACE/repo-extra.json" -u $CHACRACTL_USER:$CHACRACTL_KEY ${chacra_url}repos/${chacra_repo_endpoint}/extra/
```

Comment:
I'm doing the same thing (https://github.com/ceph/ceph-build/blob/master/nfs-ganesha/build/setup#L54) that ceph-dev-build is doing (https://github.com/ceph/ceph-build/blob/master/ceph-dev-build/build/setup_deb#L109), (https://github.com/ceph/ceph-build/blob/master/ceph-dev-build/build/setup_rpm#L117)